.github/workflows/js_tests.yml

```diff
@@ -15,12 +15,20 @@ jobs:
         node-version: [10, 12]
```

Comment:
maybe bumping the node versions might hepl? we definitelly do not need to test with nodejs 10 and we would be ok testing just on 14